deploy/docker/cp-api-preconfigure/preconfigure-api

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright 2022 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function is_dir_empty {
+    local dir_path="$1"
+
+    if [ ! -d "$dir_path" ]; then
+        mkdir -p "$dir_path"
+        return 0
+    fi
+
+    shopt -s nullglob
+    shopt -s dotglob
+
+    local dir_contents=(${1}/*)
+
+    shopt -u nullglob
+    shopt -u dotglob
+
+    local result=0
+    (( ${#dir_contents[*]} )) && result=1 || result=0
+    return $result
+}
+
+function configure_icon_win_distribution() {
```

Comment:
Right now, this method will be executed every time during `cp-api-srv` pod startup. But, the `cloud-data` app itself is not changing frequently, besides for the HA deployments such operation seems necessary during the first pod boot up only. It's not that crucial, but would be nice to reduce startup time, where possible.

It could be implemented using the following approach:
1. Create `cloud_data_win_checksum.json`  during `.exe` processing
```json
{
    "executable_sum": "<CHECKSUM_1>",
    "icon_sum": "<CHECKSUM_2>"
}
```
2. Compare values from there with the ones, calculated for the current executable and icon
3. In case JSON file exists, checksum values are matching and updated executable is present - skip icon replacement